eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -202,9 +192,21 @@ class Setup(val datadir: File,
     }
     logger.info(s"current blockchain height=$blocks")
     blockHeight.set(blocks)
-    bitcoinClient
+    (bitcoinClient, chainHash)
   }
 
+  val databases = {
+    logger.info(s"connecting to database with instanceId=$instanceId")
+    Databases.init(config.getConfig("db"), instanceId, chaindir, db)
+  }
```

Comment:
Nit: (move instantation of `instanceId)`
```suggestion
  val instanceId = UUID.randomUUID()
  logger.info(s"connecting to database with instanceId=$instanceId")
  val databases = Databases.init(config.getConfig("db"), instanceId, chaindir, db)
```

Reply:
Good idea, done in https://github.com/ACINQ/eclair/pull/2660/commits/bc7cdf109cfb11ab93590a7c74dff8e10c113634